src/Department/Department.js

```diff
@@ -11,30 +11,31 @@ class Department extends Component {
   }
 
   componentDidMount () {
-    const departmentid = this.props.match.params.id;
+    const { match: { params: { id } } } = this.props
     const getDepartamentById = async () => {
-      const data = await API.getDepartamentById(departmentid);
-      if (data.statusText === "OK") {
+      try {
+        const data = await API.getDepartamentById(id)
+        if (data.status === 200) {
+          this.setState({
+            departmentDescription: data.data.description
+          })
+        }
```

Comment:
in case there is an api error it will not pass there, so no need to check for the response status